docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+---
+id: k8s_guide
+title: A Kubernetes tutorial
+---
+# Getting Started
+:::warning[DISCLAIMER]
+This tutorial contains some parts that are CERN-specific. <br/>
+The tools utilised to set up and to run the cluster are chosen accordingly. Feel free to use your own.
+:::
+
+In order to efficiently manage the cluster and implement continuous delivery, the following tools are required.
+
+## Kubernetes: `kubectl`
+[Reference Documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/).
+## Helm Charts
+[Reference Documentation](https://helm.sh/docs/intro/install/#from-script).
+```bash
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+
+chmod 700 get_helm.sh
+
+./get_helm.sh
+```
+
+Also, add the `bitnami` repo: 
+```helm
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+So that we can search the repo with `helm search repo bitnami`.
+
+## Continuous delivery: `flux`
+[Reference Documentation](https://fluxcd.io/flux/get-started/).
+### [Install the Flux CLI](https://fluxcd.io/flux/installation/#install-the-flux-cli)
+```sh
+curl -s https://fluxcd.io/install.sh | sudo bash
+git status
+git add .
+git commit -m "first commit"
+git push
+```
+### [Bootstrap Flux](https://fluxcd.io/flux/cmd/flux_bootstrap_gitlab/)
+Then setup the flux in the gitlab repo, by exporting the secret and running the command
+```sh
+flux bootstrap gitlab \
+  --owner=<group> \
+  --repository=<repository name> \
+  --branch=<main branch> \
+  --path=<path> \
+```
+## (optional) Monitoring: `k9s`
+Get the binary from [here](https://github.com/derailed/k9s/releases) looking for the proper dist; extract, and move to `/usr/local/bin/
+
+# Setting up the CERN infrastructure
+## Creating the cluster on Openstack
+Please refer to the [documentation](https://kubernetes.docs.cern.ch/docs/getting-started/).
+It is recommended to choose a master node with a sufficient amount of memory, e.g. `m2.large`.
+
+The minimum working configuration consists of 1 master node and 5 worker nodes.
+
+## Create the database with DBOD
+1. Create a DataBase On Demand (DBOD) using the [DBOD dashboard](https://dbod.web.cern.ch/pages/dashboard);  `postgres` is highly recommended. 
+	- Have to wait for the db to be approved. Will get a mail and will receive admin credentials (to be changed) 
+2. https://github.com/vre-hub/vre/wiki/Software-components#2-database
+3. https://codimd.web.cern.ch/s/ZFIkp7PWG#2-Database-configuration
+
+Once the db creation is finalised, we can use `psql`, to set up the credentials. 
```

Comment:
Is rucioitdb just the dbod requested here? The way this is written it sounds like it's an existing resource

Reply:
Clarified.